src/Symfony/Component/PropertyInfo/Extractor/PhpStanExtractor.php

```diff
@@ -59,6 +60,10 @@ final class PhpStanExtractor implements PropertyTypeExtractorInterface, Construc
      */
     public function __construct(array $mutatorPrefixes = null, array $accessorPrefixes = null, array $arrayMutatorPrefixes = null)
     {
+        if (!class_exists(ContextFactory::class)) {
+            throw new \LogicException(sprintf('Unable to use the "%s" class as the "phpdocumentor/type-resolver" package is not installed.', __CLASS__));
```

Comment:
In exceptions like this one, we usually suggest a Composer command to run to fix the problem, don't we?